src/transport/SessionConnectionDelegate.h

```diff
@@ -35,12 +40,23 @@ class DLL_EXPORT SessionConnectionDelegate
 
     /**
      * @brief
-     *   Called when the underlying connection for the session is closed.
+     *   Called when the underlying TCP connection for the session is closed.
      *
+     * @param conn          The TCP connection state
      * @param session       The handle to the secure session
      * @param conErr        The connection error code
      */
-    virtual void OnTCPConnectionClosed(const SessionHandle & session, CHIP_ERROR conErr) = 0;
+    virtual void OnTCPConnectionClosed(const Transport::ActiveTCPConnectionState & conn, const SessionHandle & session,
+                                       CHIP_ERROR conErr) = 0;
+
+    /**
+     * @brief
+     *   Called when the underlying TCP connection attempt for the session is complete. Returns true if handled.
```

Comment:
What does "handled" mean?  What needs to be handled?  If I were implementing this API, I would have no idea whether to return true or false.

Reply:
Updated text in follow-up